cypress/e2e/dashboard.spec.ts

```diff
@@ -0,0 +1,270 @@
+      getReviewRequestButton().click()
+      cy.contains(REVIEW_MODAL_SUBTITLE).should("be.visible")
+
+      // Assert
+      getAddReviewerDropdown().click()
+      cy.contains("No options").should("be.visible")
+    })
+    it.skip("should not be able to create a review request when there are no changes", () => {
+      // Arrange
+      // Act
+      // Assert
+      throw new Error("Not implemented")
+    })
+    it("should be able to create a review request successfully", () => {
+      // Arrange
+      addAdmin(E2E_EMAIL_COLLAB.email)
+      editUnlinkedPage(
+        "faq.md",
+        "some asdfasdfasdf content",
+        E2E_EMAIL_TEST_SITE.repo
+      )
+
+      // Act
+      getReviewRequestButton().click()
+      cy.contains(REVIEW_MODAL_SUBTITLE).should("be.visible")
+      cy.get(TITLE_INPUT_SELECTOR).type(MOCK_REVIEW_TITLE)
+      // NOTE: should not allow submitting of review prior to selecting reviewer
+      getSubmitReviewButton().should("be.disabled")
+      getAddReviewerDropdown().click()
+      // select reviewer and click
+      selectReviewer(E2E_EMAIL_COLLAB.email)
+      removeReviewers()
+      getSubmitReviewButton().should("be.disabled")
+      // add back reviewer
+      getAddReviewerDropdown().click()
+      selectReviewer(E2E_EMAIL_COLLAB.email)
+      cy.get(DESCRIPTION_TEXTAREA_SELECTOR).type(MOCK_REVIEW_DESCRIPTION)
+
+      // Assert
+      getSubmitReviewButton().click()
+      cy.contains("Review request submitted").should("be.visible")
+    })
+  })
+  describe.skip("has pending review requests", () => {
+    before(() => {
+      addAdmin(E2E_EMAIL_COLLAB.email)
+      // NOTE: Create a review request if none exists
+      listReviewRequests().then((requests) => {
+        if (!requests || !requests.length) {
+          createReviewRequest(
+            MOCK_REVIEW_TITLE,
+            [E2E_EMAIL_COLLAB.email],
+            MOCK_REVIEW_DESCRIPTION
+          )
+        }
+      })
+    })
+    it("should have the pending review alert on the workspace", () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it("should prevent the requestor from approving their own review request", () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it('should show the review request on the workspace with the tag "Pending review"', () => {
+      // Arrange
+      // Act
+      // Assert
+    })
+    it("should have changes reflected in the currently open review request", () => {
+      // Arrange
+      // TODO: If this is hard, maybe we can just do delete + add content + have new page
+      // NOTE: We define changes as being
+      // 1. Adding a new page
+      // 2. Editing an existing page
+      // 3. Deleting an existing page
+      // 4. Rename an existing page
+      // 5. Moving an existing page
+      // Act
+      // Assert
+    })
+    it("should be able to have the request approved as a collaborator", () => {
```

Comment:
[Non-nit] we should have two tests here to test critical access control flows, ie 
'contributor should not be able to approve the request' 
'admin should be able to approve the request'

Reply:
contributors won't be able to be added as a reviewer in the first place! there's a test for this

```ts
    it("should not be able to create a review request without a reviewer", () => {
      // Arrange
      cy.createEmailUser(
        E2E_EMAIL_COLLAB.email,
        USER_TYPES.Email.Collaborator,
        USER_TYPES.Email.Admin
      )
      addCollaborator(E2E_EMAIL_COLLAB.email)

      // Act
      // NOTE: There should be at least 1 other admin to be able to create a review request
      getReviewRequestButton().click()
      cy.contains(REVIEW_MODAL_SUBTITLE).should("be.visible")

      // Assert
      getAddReviewerDropdown().click()
      cy.contains("No options").should("be.visible")
    })
```

Reply:
Wait in that test case, we dont see any reviewers as an option because I cannot self review right? This is not the same as checking that collab cannot be a reviewer? 

seems like to fully test that flow we might need another user email here (which might be out of scope of the pr) but it feels like a critial access flow control we want to add e2e tests for. Wdyt?

Reply:
this is the creation of a review request - the review request has **not** been created yet. the test case for the creation one is the below one, where there is only 1 admin (the creator of the RR) and hence, the RR cannot be created

```ts
should not be able to create a review request when the site has 1 collaborator
```

in this one, we add a collaborator (now there's 2 site members, admin + collab) and the admin tries to create a RR but fails because there's only 1 other collab that cannot be a reviewer